### PR TITLE
New package: UZ7HO.EasyTerm version 4.9

### LINKED ISSUE #115468: [Package Request]: EasyTerm
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [X] The installer meets the above requirements

### Please provide the following information

Download Page Url: http://uz7.ho.ua/packetradio.htm
Publisher: UZ7HO
Package Name: EasyTerm
Description: Windows-based program designed for AX.25 Packet amateur radio communications.
Package Version: 4.9
Installer URL: http://uz7.ho.ua/apps/easyterm49.zip

### PR DESCRIPTION
**Pull request by Dragon1573**
## Description

- Resolve https://github.com/microsoft/winget-pkgs/issues/115468

## Checks

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/123419)

### CODE REVIEW

File: manifests/u/UZ7HO/EasyTerm/4.9/UZ7HO.EasyTerm.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+InstallerType: zip
+Commands:
+- EasyTerm
+Installers:
+- Architecture: x86
+  NestedInstallerType: portable
+  NestedInstallerFiles:
+  - RelativeFilePath: Term.exe
+    PortableCommandAlias: EasyTerm
+  InstallerUrl: http://uz7.ho.ua/apps/easyterm49.zip
```
Comment (Dragon1573) [inline manifests/u/UZ7HO/EasyTerm/4.9/UZ7HO.EasyTerm.installer.yaml:15]:
```suggestion
  InstallerUrl: https://uz7.ho.ua/apps/easyterm49.zip
```

**Comment by Dragon1573** [comment]
This site is personal hosted and seems not configured its own SSL/TLS certificate for HTTPS connection. The certificate is authorized to `s2.ho.ua` by **Let's Encrypt**.

**Comment by stephengillie** [comment]
Hi @Dragon1573,

This ISV's host is misconfigured. They're using the `s2` subdomain certificate to secure the `uz7` subdomain. This is why the SSL error is seen. 
image

**Comment by Dragon1573** [comment]
- Related https://github.com/microsoft/winget-pkgs/pull/123419#issuecomment-1774115636

It is expected. Error is caused by the package sote itself.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**